lite/examples/image_segmentation/raspberry_pi/segment.py

```diff
@@ -112,17 +122,15 @@ def run(model: str, display_mode: str, num_threads: int, enable_edgetpu: bool,
 
 def visualize(input_image: np.ndarray, segmentation_map_image: np.ndarray,
               display_mode: str, fps: float,
-              colored_labels: List[ColoredLabel]) -> np.ndarray:
+              colored_labels: List) -> np.ndarray:
```

Comment:
You should import the ColoredLabel type in Segmentation so that this have type hint.

Reply:
This problem has been resolved